src/tests/class_tests/openms/source/FASTAFile_test.cpp

```diff
@@ -180,15 +216,76 @@ START_SECTION([EXTRA] test_strange_symbols_in_sequence)
   data.push_back(temp_entry);
   data.push_back(temp_entry); // twice
 
+
   file.store(tmp_filename, data);
   file.load(tmp_filename, data2);
-  
+
   ABORT_IF(data2.size() != 2);
   TEST_EQUAL(data2[0] == temp_entry, true);
   TEST_EQUAL(data2[1] == temp_entry, true);
-
 END_SECTION
 
+
+START_SECTION(test_white_spaces)
+//test if spaces and tabulators are removed correctly
+    String tmp_filename;
+    NEW_TMP_FILE(tmp_filename);
+    FASTAFile file;
+    vector<FASTAFile::FASTAEntry> data, data2;
+
+                FASTAFile::FASTAEntry temp_entry;
+                temp_entry.identifier = String("P68509|1433F_BOVIN");
+                temp_entry.description = String("This is the description of the first protein");
+                temp_entry.sequence = String("GDREQLLQRAR LAEQ\tAERYDDMASAMKAVTEL");
+                data.push_back(temp_entry);
+
+                file.store(tmp_filename, data);
+                file.load(tmp_filename, data2);
+
+                ABORT_IF(data2.size() != 1);
+                TEST_EQUAL(data2[0].sequence == string("GDREQLLQRARLAEQAERYDDMASAMKAVTEL"), true);
+
+        END_SECTION
+
+        START_SECTION([EXTRA] test_position) //
+                // test if setPosition() works correctly
+                String tmp_filename;
+                NEW_TMP_FILE(tmp_filename);
+                FASTAFile file;
+
+                vector<pair<streampos, FASTAFile::FASTAEntry>> data1;
+                vector<pair<streampos, FASTAFile::FASTAEntry>> data2;
+                FASTAFile::FASTAEntry temp_entry;
+                file.readStart(OPENMS_GET_TEST_DATA_PATH("FASTAFile_test.fasta"));
+
+
+                for (int i=0; i<5; i++){
+                file.readNext(temp_entry);
+                data1.push_back(std::make_pair(file.position(), temp_entry));
+                }
+                file.setPosition(data1[0].first);
+                for(int i=0; i<3; i++){
+                    file.readNext(temp_entry);
+                    data2.push_back(std::make_pair(file.position(), temp_entry));
+                }
+
+                ABORT_IF(data1.size() != 4 || data2.size() != 3 );
+
+                for(int i=1; i<=data1.size(); i++) {
+                    TEST_EQUAL(data1[i].second.identifier == data2[i - 1].second.identifier, true);
```

Comment:
nice.

Did you try `TEST_EQUAL(x, y)` instead of `TEST_EQUAL(x==y, true)`?
It has the advantage that the error message is better if it ever fails.